tools/src/main/scala/com/nvidia/spark/rapids/tool/qualification/PluginTypeChecker.scala

```diff
@@ -52,12 +61,74 @@ class PluginTypeChecker {
     writeFormats = writeFormatsTest
   }
 
+  def setOperatorScore(filePath: String): Unit = {
+    val source = Source.fromFile(filePath)
+    supportedOperatorsScore = readSupportedOperators(source).map(x => (x._1, x._2.toInt))
+  }
+
+  def setSupportedExecs(filePath: String): Unit = {
+    val source = Source.fromFile(filePath)
+    supportedExecs = readSupportedOperators(source)
+  }
+
+  def setSupportedExprs(filePath: String): Unit = {
+    val source = Source.fromFile(filePath)
+    supportedExprs = readSupportedOperators(source)
```

Comment:
make comments here and in the Execs functions that we are only reading the first 2 columns for now, other columns ignored intentionally